chrome_manifest_v3/class_app.js

```
@@ -0,0 +1,869 @@
+
+  /***
+   * Middle-truncate a string
+   */
+  midTruncate(text, max, add) {
+    const retn = text || '';
+    const add_k = this.decodeEntities(add || '');
+    const max_k = Math.abs((max || 0) - add_k.length);
+    const mid_k = (max_k + 0.01) / 2;
+
+    if (text && text.length > max_k) {
+      retn = text.slice(0, mid_k + 1) + add_k + text.slice(-mid_k);
```

Biome (1.9.4)

[error] Can't assign retn because it's a constant

This is where the variable is defined as constant

Unsafe fix: Replace const with let if you assign it to a new value.


(lint/correctness/noConstAssign)